src/tests/jsonLogic.test.js

```diff
@@ -243,6 +244,23 @@ describe('jsonLogic: cross-values validations', () => {
     });
   });
 
+  // TODO: Implement this test.
+  describe.skip('Reduce', () => {
```

Comment:
> rebased with main @dragidavid / @thien-remote with the move to next I'll leave this in your capable hands to ensure this is eventually moved into there. Feel free to merge/disregard this PR!

I was checking this now, I bet it got missed by David or Thien. All good, I'm defaulting to action in merging this now (as it's only a new test case, nothing changed.) The team will take care of it as part of [this internal issue](https://linear.app/remote/issue/DEVXP-2547)

cc @lukad